superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -314,15 +314,7 @@ export function transformSeries(
             opacity: opacity * areaOpacity,
           }
         : undefined,
-    emphasis: {
-      // bold on hover as required since 5.3.0 to retain backwards feature parity:
-      // https://apache.github.io/echarts-handbook/en/basics/release-note/5-3-0/#removing-the-default-bolding-emphasis-effect-in-the-line-chart
-      // TODO: should consider only adding emphasis to currently hovered series
-      lineStyle: {
-        width: 'bolder',
```

Comment:
@villebro is it ok to remove this?

Reply:
> I also removed the line boldness on hover because on dashed lines the animation is a bit distracting.

@eschutho I believe this was added to keep the hover behavior consistent with other chart types.